go.mod

```diff
@@ -45,8 +45,13 @@ require (
 	sigs.k8s.io/testing_frameworks v0.1.2 // indirect
 )
 
+replace github.com/codeready-toolchain/api => github.com/matousjobanek/api v0.0.0-20200106115125-b76764e1418b
```

Comment:
I guess this can be removed, now that https://github.com/codeready-toolchain/api/pull/82 was merged

Reply:
Done 191e29d